### PR TITLE
Fixed validation rules to disallow lookups stored as helm variables

### PR DESCRIPTION
**Pull request by 5herlocked**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by elamaran11 — APPROVED** [review]
LGTM

File: src/services/validate.ts [hunk 1]
```diff
@@ -187,7 +187,7 @@ export default class ChartValidatorService extends BaseService {
     // Find any instance of "lookup" that starts with an opening parenthesis and ignore any white spaces between opening
     // and the word itself
     // eslint-disable-next-line no-useless-escape
-    const grepLookup = spawnSync('grep', ['-Rilne', '"\{\{-?\s*lookup"', `"${this.toValidate}"`], {shell: true, encoding: "utf8"});
+    const grepLookup = spawnSync('grep', ['-Rilne', '"\{\{-?\s*.*lookup"', `"${this.toValidate}"`], {shell: true, encoding: "utf8"});
```
Comment (jcabrerizo) [inline src/services/validate.ts:190]: Expecting at the word `lookup` with and space after would prevent false positives on expressions like:
```
...
{{ include "my.secrets.lookup" (dict "namespace" .Release.Namespace "secret" "my-controller-secret" "key" "ssl-cert.key" "defaultValue" $cert.Key) }}
...
```

and still capturing it on 
```
{{- $secretData := (lookup "v1" "Secret" .namespace .secret).data  -}}
```

```suggestion
    const grepLookup = spawnSync('grep', ['-Rilne', '"\{\{-?\s*.*lookup\s"', `"${this.toValidate}"`], {shell: true, encoding: "utf8"});
```

Comment (5herlocked) [reply src/services/validate.ts]: resolved, also made lookup a capture group.

**Review by jcabrerizo — APPROVED** [review]
Hi @5herlocked, I tested this and the `lookup` is working. Thanks!

**Review by elamaran11 — APPROVED** [review]

**Pull request merged**